packages/cache/RELEASES.md

```diff
@@ -8,4 +8,7 @@
 - Fixes issues with the zstd compression algorithm on Windows and Ubuntu 16.04 [#469](https://github.com/actions/toolkit/pull/469)
 
 ### 0.2.1
-- Fix to await async function getCompressionMethod
\ No newline at end of file
+- Fix to await async function getCompressionMethod
+
+### 1.0.0
+- Downloads Azure-hosted caches using the Azure SDK for speed and reliability
```

Comment:
Could we list the breaking changes here?

Reply:
Done.  Are there any other breaking changes we would want to make now?

Reply:
I'm working on a follow-on PR to display progress.  Let's not release 1.0.0 just yet.

Reply:
@joshmgross Here's the follow-on PR - https://github.com/dhadka/toolkit/pull/2